pkg/update_pod_status_util.go

```diff
@@ -82,10 +82,10 @@ func execute(cmd []string, stderr *os.File) error {
 	return command.Run()
 }
 
-func (initp *InitParams) getPodImage(pod *corev1.Pod) string {
+func (initp *InitParams) getPodImages(pod *corev1.Pod) (serverImage, initImage string) {
 	initp.logger.Info("Get pod image", "podName", pod.Name)
 
-	return pod.Spec.Containers[0].Image
+	return pod.Spec.Containers[0].Image, pod.Spec.InitContainers[0].Image
```

Comment:
Are we sure that the first init-container will always be ako-init?

Reply:
while creating a sts object in the operator, like the server container it appends the podspec containers after the ako init container.